package/system/procd/patches/0001-initd-init-add-minimal-SELinux-policy-loading-suppor.patch

```diff
@@ -0,0 +1,110 @@
+From fe74ad8b11977d0ced5c44f5e389c50ee70bc008 Mon Sep 17 00:00:00 2001
+From: Thomas Petazzoni <thomas.petazzoni at bootlin.com>
+Date: Thu, 23 May 2019 13:57:30 +0200
+Subject: [PATCH] initd/init: add minimal SELinux policy loading support
+
+In order to support SELinux in OpenWRT, this commit introduces minimal
+support for loading the SELinux policy in the init code. The logic is
+very much inspired from what Busybox is doing: call
+selinux_init_load_policy() from libselinux, and then re-execute init
+so that it runs with the SELinux policy in place and enforced.
+
+Signed-off-by: Thomas Petazzoni <thomas.petazzoni at bootlin.com>
```

Comment:
@tpetazzoni I think this does not count as a valid email address as it breaks `git send-email` auto CCing. Is the usage of ` at ` rather than `@` important to you? I'd figure that most spam bots are smart enough to figure out that `<.+at.+.com>` is an email address.

Reply:
I checked the Linux Kernel and it doesn't seem to be used for final SoB lines, however as I corrected Open**WRT** and thereby added myself as final SoB with a correct mail address, I'm figuring it's fine.

Reply:
I don't know what replaced @ by "at", but it's certainly not intentional, and probably something done by one of the mailing list software on the way. You can and should definitely use my real e-mail address with an @ sign. Thanks!

Reply:
Bad news, `procd` now contains your *spam proof email*. We'll fix that for the other cases